module_1/assignment1_2.html

```diff
@@ -0,0 +1,10 @@
+<DOCTYPE html>
+<html>
+<head>
+        <script src = "myscript_1.js"></script>
```

Comment:
Have to observe when happen with the body content when we pass file in script ? Please add a short comment.